rpcs3/CMakeLists.txt

```diff
@@ -107,7 +107,11 @@ if(USE_NATIVE_INSTRUCTIONS)
 endif()
 
 if(NOT MSVC)
-	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-invalid-pch -Wno-unused-command-line-argument -fexceptions")
+	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-invalid-pch -fexceptions")
+	if(${CMAKE_CXX_COMPILER_ID} MATCHES "Clang")
```

Comment:
This is a case of double expansion, drop the ${} around the variable name.

Reply:
Fixed